tools/auth_keys_generator.go

```diff
@@ -26,17 +26,22 @@ func main() {
 	//Generate keys in JWK format
 	pubJSJWKS, privJSJWKS, kid, _ := keygen_tools.GenJSJWKS(priv, pub)
 
-	tokenString, err := getTokenString(getUserToken(), kid, priv)
```

Comment:
where do we use this file? i found that there is a function generate_keys in hack/generate.sh, but it does not seems that we call it anyware ( at least i have not found :) )

Reply:
iinm, it's invoked by 'generate-keys' in deploy-test: https://github.com/openshift/assisted-service/blob/master/Makefile#L345
using this magic target:) https://github.com/openshift/assisted-service/blob/master/Makefile#L149

Reply:
yes, thanks @danielerez